### PR TITLE
Migrate Fabric's installation to use the v2 API

### PR DESCRIPTION
**Pull request by ghost**
This pull request solves #475 by:
- Using the API's profile JSON instead of using Technic's
- Dropping the usage of Fabric's Yarn in favor of directly using the game version
- Determining what version is stable or a snapshot by using the same API as above

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]

File: src/common/api.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ export const getForgeManifest = () => {
 };
 
 export const getFabricManifest = () => {
-  const url = `${FABRIC_APIS}/v2/versions?timestamp=${new Date().getTime()}`;
+  const url = `${FABRIC_APIS}/versions?timestamp=${new Date().getTime()}`;
```
Comment (modmuss50) [inline src/common/api.js]: What about removing the timestamp, there is no need for this. it just puts more load on the meta api, it already has a short cache time of 60 seconds.

Comment (ghost) [reply src/common/api.js]: There weren't no timestamps (on the three manifests, including this one) until [this commit](https://github.com/gorilla-devs/GDLauncher/commit/89eb7c4e66024bc4bbd88ea0e05d6a5de219ce26), it looks like that without them, the manifests were cached.

Comment (modmuss50) [reply src/common/api.js]: They should be cached, I see no reason for them not to be cached.

Comment (ghost) [reply src/common/api.js]: Alright, I'm removing the timestamp from the request for Fabric's manifest. I'm leaving the other 2 for someone else, I don't want to break anything silently.

**Comment by blarfoon** [comment]
The timestamp has been added for forge, since I noticed it wasn't updating fast enough when a new version came out. We can try to remove it from fabric since you said it has a low caching time

**Comment by blarfoon** [comment]
LGTM. We also need to implement a config migrator to adapt previous fabric installations to this new method, but I'll write that myself since it's a little bit more non-trivial. Thank you for the PR :)

**Pull request merged**